examples/onlineboutiquemicroservice/README.md

```diff
@@ -0,0 +1,59 @@
+# Online Boutique Microservices Example
+
+## Overview
+
+Online Boutique Microservices example is based on the [cloud-native microservices demo application](https://github.com/GoogleCloudPlatform/microservices-demo) of [Google Cloud Platform](https://github.com/GoogleCloudPlatform). 
+The original example consists of 10 microservices implemented in various languages and 3 of those microservices are implemented in Ballerina here.
+This example shows how ballerina gRPC module can work effectively with microservices implemented in other languages.
```

Comment:
```suggestion
This example shows how Ballerina gRPC module can work effectively with microservices implemented in other languages.
```

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/392/commits/56b949cc1bb433c00a38ac181b0b233f12087e9b